### ISSUE TITLE
Installing the devkit creates an error during the "Building Cosmos" stage

### ISSUE THREAD

**Issue by 365Tito** [CosmosOS/Cosmos#790]
When running the install-VS2017.bat from my Administrator account, everything goes correctly untill the "Building Cosmos" stage of the installation.
I've taken the necessary precautions to:
1. Install the .NET Framework 2.0 SDK and runtime
2. Have the folders in the correct order:
SOURCE folder, contains:
>Cosmos-master
>IL2CPU
>XSharp

Link to the log tail:
https://pastebin.com/YAGwhePx

The error, which is posted in the above pastebin link mostly reports having issues with targeting a .NET Framework 4.6.2 from a .NET Standard 2.0 Project. I have tried playing around with the project properties of the Build/Builder etc, but to no avail...

I would appreciate your help!
Thanks in advance!

**Comment by fanoI** [comment]
Please read here: https://github.com/CosmosOS/Cosmos/blob/master/SUPPORT.md

**Comment by jp2masa** [comment]
@365Tito Rename "Cosmos-master" to "Cosmos".

**Comment by 365Tito** [comment]
Thank you for the rather quick replies ! 
Tried both solutions... 
@fanoI - I already have 4.6.2 installed. The installer only offered the (Repair / Uninstall) options.
@jp2masa - I tried that too... it doesn't work.

https://docs.microsoft.com/en-us/dotnet/standard/net-standard

This table right here , interestingly enough shows that the 4.6.2 .NET Framework does not support .NET Standard 2.0, which I guess, is further proved here:

https://github.com/dotnet/standard/blob/master/docs/versions/netstandard2.0.md

And... what now?
